packages/react-components/react-progress/library/src/components/ProgressBar/useProgressBarStyles.styles.ts

```diff
@@ -41,16 +37,16 @@ const useRootStyles = makeStyles({
     },
   },
   rounded: {
-    borderRadius: tokens.borderRadiusMedium,
+    borderRadius: semanticTokens.ctrlProgressCorner,
   },
   square: {
-    borderRadius: tokens.borderRadiusNone,
+    borderRadius: semanticTokens.cornerZero,
   },
   medium: {
-    height: barThicknessValues.medium,
+    height: semanticTokens.ctrlProgressHeightFilled,
   },
   large: {
-    height: barThicknessValues.large,
+    height: semanticTokens.ctrlProgressLgHeightFilled,
```

Comment:
Since this is under root styles we should probably use `ctrlProgressLgHeightEmpty`/`ctrlProgressLgHeightEmpty` to be more semantically correct.

We should probably add ctrlProgressHeightFilled to the color portion of the progress bar, but have it fall back to `100%` to match the existing structure.